public/app/features/alerting/unified/components/mute-timings/useMuteTimings.tsx

```diff
@@ -294,3 +297,19 @@ export const useValidateMuteTiming = ({ alertmanager }: BaseAlertmanagerArgs) =>
       });
   };
 };
+
+/**
+ * @deprecated This will be deprecated by the K8S API.
+ * Once that is enabled by default, this should be removed and the `getMuteTimingList` should be removed
+ */
+export const useSelectableMuteTimings = ({ alertmanager }: BaseAlertmanagerArgs) => {
+  const useK8sApi = shouldUseK8sApi(alertmanager);
+
+  const fetchGrafanaMuteTimings = useGetMuteTimingListQuery(undefined, {
```

Comment:
this naming is confusing for me, as I see that when we are using k8s api we use the naming 'Grafana' , and when we use alertmanager api we use the naming alertmanager. 
I would expect `fetchGrafanaMuteTimings` to be something like `fetchAlertManagerMuteTimings` ? or is something I'm missing here?

Reply:
I've changed the naming and the logic - I think I was previously calling the wrong API if we hadn't enabled the k8s API - what do you think of it now?